pkg/cluster/operation/check.go

```diff
@@ -175,17 +180,31 @@ func checkOSInfo(opt *CheckOptions, osInfo *sysinfo.OS) *CheckResult {
 			return result
 		}
 	case "amzn":
+		// https://aws.amazon.com/linux/amazon-linux-2023/
+		if osInfo.Version == "2023" {
+			return result
+		}
+
 		// Amazon Linux 2 is based on CentOS 7 and is recommended for
 		// AWS Graviton 2 (ARM64) deployments.
+		// https://aws.amazon.com/amazon-linux-2/
 		if ver, _ := strconv.ParseFloat(osInfo.Version, 64); ver < 2 || ver >= 3 {
-			result.Err = fmt.Errorf("%s %s not supported, use version 2 please",
+			result.Err = fmt.Errorf("%s %s not supported, use Amazon Linux 2 or Amazon Linux 2023 please",
 				osInfo.Name, osInfo.Release)
 			return result
 		}
-	case "centos", "redhat", "rhel", "ol":
-		// check version
-		if ver, _ := strconv.ParseFloat(osInfo.Version, 64); ver < 7 {
-			result.Err = fmt.Errorf("%s %s not supported, use version 8 please",
+	case "centos":
+		// CentOS Linux is EOL
+		// CentOS Stream 9 and newer is still fine
+		if ver, _ := strconv.ParseFloat(osInfo.Version, 64); ver < 9 {
```

Comment:
Looks a little newer to me, maybe `7.3` or so? Or mark `result.warn`?

Reply:
You think we should still list CentOS Linux 7.3 as supported?

Reply:
> You think we should still list CentOS Linux 7.3 as supported?

I just worry existed tiup deployed on `CentOS7` will error.